core/src/OpenACC/Kokkos_OpenACC_Team.hpp

```diff
@@ -338,6 +338,11 @@ class TeamPolicyInternal<Kokkos::Experimental::OpenACC, Properties...>
         m_chunk_size(0) {
     init(league_size_request, team_size_request, 1);
   }
+
+  TeamPolicyInternal(const PolicyUpdate, const TeamPolicyInternal& other,
+                     Kokkos::Experimental::OpenACC)
+      : TeamPolicyInternal(other) {}
```

Comment:
Noting that the team policy does not store its exec space instance.
cc @seyonglee

Reply:
I noticed, and did not wanted to slow down this PR with such weird things :jack_o_lantern: 

But it clearly needs to be addressed.